### PR TITLE
Online Booking feedback form

### PR DESCRIPTION
**Pull request by benbarnett**
Progressively enhanced to keep the user in the same place they
were when javascript is available.

Creates a Zendesk ticket in the format:

```
Name: Customer name
Email: Customer email

Message: Customer comments
```

### CODE REVIEW

File: app/forms/booking_feedback_form.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class BookingFeedbackForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :email, :message
+
+  validates :name, presence: true
+  validates :email, presence: true, format: { with: /.+@.+\..+/ }
+  validates :message, presence: true
+
+  def initialize(opts)
```
Comment (benlovell) [inline app/forms/booking_feedback_form.rb]: `opts = {}` then you could remove the `{}` args at every callsite.

File: app/controllers/feedback_controller.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    params
+      .fetch(:booking_feedback, {})
+      .permit(
+        :name,
+        :email,
+        :message
+      )
+  end
+
+  def feedback_comment
```
Comment (benlovell) [inline app/controllers/feedback_controller.rb]: Use a squiqqly heredoc here to tidy this up perhaps.

Comment (benbarnett) [inline app/controllers/feedback_controller.rb]: Diddly done

**Comment by benlovell** [comment]
This totes needs tests, but we can merge for now and I'll make sure I circle back around when the opportunity arises.

**Comment by benlovell** [comment]
:shipit:

**Pull request merged**